crates/bevy_gltf/src/loader.rs

```diff
@@ -262,7 +263,26 @@ async fn load_gltf<'a, 'b>(
             });
         })
         .into_iter()
-        .filter_map(|result| result.ok())
+        .filter_map(|res| {
+            if let Err(err) = res.as_ref() {
+                match err {
+                    GltfError::AssetIoError(error) => {
+                        warn!("Error loading GLTF texture: {}", error)
+                    }
+                    GltfError::ImageError(error) => {
+                        warn!("Error loading GLTF texture, maybe you're missing a feature for the file format: {}", error)
+                    }
+                    GltfError::InvalidImageMimeType(error) => {
+                        warn!("Error loading GLTF texture: {}", error)
+                    }
+                    GltfError::Base64Decode(error) => {
+                        warn!("Error loading GLTF texture: {}", error)
+                    }
+                    _ => (),
```

Comment:
Why are we doing a custom message per error?
`GltfError` should implement `Debug`.
```rust
warn!("GltfError: {:?}", err);
```

Reply:
because I tried being clever and only logging relevant errors but that actually doesn't matter as everything will be properly logged anyway 👍